### PR TITLE
lines 72-98 i want to add a text output and action button

### PR DESCRIPTION
**Pull request by ADarnellF**
@cansavvy Hello, 
i wanted to see if you could give run this code and explain where i would add an " action button" 
so that way can alumni go and update their information, they can clicking  submit!

### CODE REVIEW

**Comment by cansavvy** [comment]
Taking a look at this @ADarnellF ! Sorry for the delay!

File: README.md [hunk 1]
```diff
@@ -8,23 +8,6 @@ output:
 runtime: shiny
 ---
 
-Inputs {.sidebar}
```
Comment (cansavvy) [inline README.md:11]: I'm a little confused on how to interpret these file changes. The README doesn't appear to be a [README in a traditional sense](https://www.mygreatlearning.com/blog/readme-file/). 

Are these changes supposed to be in the `Alumni` file? If so, should that file have the `.Rmd` suffix? Last I knew the file you were working from was an R Markdown. 


I would suggest putting instructions in the README about how to run the code that is in the `Alumni.Rmd` file. 
Aka, is it meant to be run from RStudio? Then click Run in RStudio? README's are super great when they give people the starting point of how to use your code. 

Let me know if this helps and/or makes sense.

**Review by cansavvy — COMMENTED** [review]
Hi @ADarnellF I have some basic questions about the file set up here and how to run your code before I dive in to your main questions. 

Sorry for the delay and thanks!

**Comment by cansavvy** [comment]
@ADarnellF we did some pair coding and figured out some things! 

- The README has a set up you might want to fill out more with start up info about this project. 
- You uploaded the most recent Rmd "dataset_update.Rmd" This looks like it runs well! But the duplicate ids might be what's throwing it off, might want to change that in the Googlesheet and see if that fixes some of the problem. 

The goal is to have the `Update information` button update the info a person enters in the Googlesheet, so let me know how it goes after you update the IDs to be unique.

**Comment by ADarnellF** [comment]
Hey candace, Thank you for these notes!
so ive added an unique code to my google sheet Successfully. The next steps would be:
allowing clients to update their information:

Problem: the action button is not working

**Comment by cansavvy** [comment]
Hey @ADarnellF Did you mean to close this pull request without merging it?

**Comment by ADarnellF** [comment]
No i wanted to make sure you saw my comments and i pushed my latest work

So i must of made a mistake? 

So the steps would be to merge it?

**Comment by cansavvy** [comment]
> No i wanted to make sure you saw my comments and i pushed my latest work
> 
> So i must of made a mistake? 
> 
> So the steps would be to merge it?
> 
> 
> 
> 
> 
> > On Jun 10, 2022, at 2:37 PM, Candace Savonen ***@***.***> wrote:
> > 
> > ﻿
> > Hey @ADarnellF Did you mean to close this pull request without merging it?
> > 
> > —
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> > You are receiving this because you were mentioned.
> 


Yeah the idea is we get the changes to the point that we like them (and the button is working) and then merge it.

**Comment by cansavvy** [comment]
I'd recommend merging this and then we can start with a fresh pull request (PR). @ADarnellF If you want to meet and discuss what I mean by this, happy to do so.

**Pull request merged**